src/components/variant/ui/drawer-note/drawer-note.tsx

```diff
@@ -74,33 +107,46 @@ const DrawerNoteModal = observer(({ close }: any) => {
 
   return (
     <div
-      ref={ref}
-      className="w-96 bg-blue-light flex flex-col py-5 px-4 rounded-xl"
+      ref={wrapperRef}
+      className={classNames(styles['modal-wrapper'], 'bg-blue-light')}
     >
-      <span className="w-full">
-        <span>{t('variant.notesFor')} </span>
+      <div className={styles['modal-title']}>
+        <div>
+          <span>{t('variant.notesFor')} </span>
+
+          <span className="text-blue-bright">
+            {`[${genInfo}] `}
+
+            <span dangerouslySetInnerHTML={{ __html: hg19 }} />
+          </span>
+        </div>
 
-        <span className="text-blue-bright">
-          {`[${genInfo}] `}
+        <Icon
+          name="Close"
+          onClick={close}
+          size={16}
+          className="cursor-pointer"
+        />
+      </div>
 
-          <span dangerouslySetInnerHTML={{ __html: hg19 }} />
-        </span>
-      </span>
-      <div className="relative mt-3">
+      <div>
         {error && (
           <div className="absolute -top-2.5 text-12 text-red-secondary">
             {error}
           </div>
         )}
 
         <textarea
-          placeholder="Enter text"
-          className="w-full mt-2 p-3 h-80 rounded-lg resize-none mx-auto"
-          rows={15}
+          ref={textareaRef}
+          placeholder={t('variant.textAboutSomething')}
           value={value}
-          onChange={(e: ChangeEvent<HTMLTextAreaElement>) =>
-            handleChange(e.target.value)
+          onChange={(event: ChangeEvent<HTMLTextAreaElement>) =>
```

Comment:
better pass a function without creating anonymous one and grab the value inside of it

Reply:
Fixed it